app/helpers/funding_helper.rb

```diff
@@ -18,11 +18,18 @@ def format_currency(amount)
     )
   end
 
+  def funding_csv_export_path(funding_type, organisation)
```

Comment:
I wonder whether we can move this method in to the `Funding::NavigationView`? That's where the other methods dealing with the shared view differences between users and system_admins live. Maybe it needs renaming..

This file seems to be more to do with the actual 'add trainee funding section' of the app.